tests/e2e/tracer/handlers/sync_async_capture.py

```diff
@@ -0,0 +1,23 @@
+import asyncio
+from uuid import uuid4
+
+from aws_lambda_powertools import Tracer
+from aws_lambda_powertools.utilities.typing import LambdaContext
+
+tracer = Tracer()
+
+
+@tracer.capture_method
+def get_todos():
+    return [{"id": f"{uuid4()}", "completed": False} for _ in range(5)]
+
+
+@tracer.capture_method
+async def async_get_users():
+    await asyncio.sleep(1)
```

Comment:
why do we need sleep here?

Reply:
I'll add a comment. It's to simulate a coroutine to block and be successfully traced. I had false positives in the past where the async code ran immediately when in fact it only traced a coroutine to be primed, not actually executed.